data_safe_haven/resources/workspace/ansible/desired_state.yaml

```diff
@@ -307,12 +297,12 @@
           systemctl stop clamav-freshclam && freshclam && systemctl start clamav-freshclam
         creates: '/var/lib/clamav/main.{c[vl]d,inc}'
 
-    - name: Install ClamAV services and timer
+    # This will create directories if src or dest ends in '/'
+    - name: Copy ClamAV services directory
       ansible.builtin.copy:
-        src: "{{ item }}"
+        src: "etc/systemd/system/"
         dest: /etc/systemd/system/
         mode: '0644'
-      with_fileglob: 'etc/systemd/system/clamav*'
       notify: Systemd daemon reload
```

Comment:
This one is fine for now, as there are no other systemd units but we should revert it if we add more.

This would copy all units and not just the clamav units.

Reply:
Yes - I guess if we add more we should change the name of the task.

Reply:
Could be part of https://github.com/alan-turing-institute/data-safe-haven/issues/2096